pkg/controllers/job/plugins/svc/const.go

```diff
@@ -19,6 +19,10 @@ package svc
 const (
 	// ConfigMapTaskHostFmt key in config map
 	ConfigMapTaskHostFmt = "%s.host"
+	// EnvTaskHostFmt is the key for host list in environment
+	EnvTaskHostFmt = "VC_%s_HOSTS"
```

Comment:
I suggest we change to volcano full name from usability view. Abbreviate is hard to remember.

Reply:
no, let's keep `VC_`.

Reply:
ok, you are the boss you decide.